src/GitHub/GitHubRepositoryParser.php

```diff
@@ -0,0 +1,32 @@
+<?php
+declare(strict_types=1);
+
+namespace Leviy\ReleaseTool\GitHub;
+
+use function explode;
+use function str_replace;
+
+class GitHubRepositoryParser
+{
+    public function getOwner(string $url): string
+    {
+        return $this->parseUrl($url)[0];
+    }
+
+    public function getRepository(string $url): string
+    {
+        return $this->parseUrl($url)[1];
+    }
+
+    /**
+     * @param string $url
+     *
+     * @return string[]
+     */
+    private function parseUrl(string $url): array
+    {
+        $url = str_replace(['git@github.com:', '.git'], '', $url);
```

Comment:
Should we throw an exception if the remote URL does not conform to this format? I.e. if it is an HTTP(S) URL?